src/ui/theme/components/button.ts

```diff
@@ -61,30 +74,52 @@ const variantSolid = (getColor: GetColor) => (
   };
 };
 
-const variantToggle = (getColor: GetColor) => (
+const variantSettings = (getColor: GetColor) => (
   props: React.ComponentProps<typeof Button>
 ) => {
-  const bgColor = getColor('ui.white', 'gray.800', 'ui.white');
-  const color = getColor('gray.800', 'ui.white', 'gray.800');
+  const { bgColor, value } = props;
+
+  const color = getColor('ui.black', 'ui.white', 'ui.black');
+
+  const checkedBgColor = getColor(
+    'ui.gray.light-warm',
+    'ui.gray.x-dark',
+    'ui.sepiaChecked'
+  );
+
   return {
     ...variantSolid(getColor)(props),
-    bgColor,
+    bgColor: getColor('ui.white', 'ui.black', 'ui.sepia'),
+    border: '1px solid',
+    borderBottom: (value === 'paginated' || value === 'scrolling') && 'none',
+    borderRadius:
+      value === 'paginated'
+        ? '0 0 0 4px'
+        : value === 'scrolling'
+        ? '0 0 4px 0'
+        : null,
```

Comment:
A couple more places where we care checking the value that I'd like to remove and instead have at the point the button is rendered

Reply:
Removed!